built_json_generator/test/built_json_generator_test.dart

```diff
@@ -175,6 +177,91 @@ class _$ValueSerializer implements StructuredSerializer<Value> {
 ''');
     });
 
+    test('generates correct serializer for built_value with nullables',
+        () async {
+      expect(
+          await generate(r'''
+library value;
+
+import 'package:test_support/test_support.dart';
+
+abstract class Value implements Built<Value, ValueBuilder> {
+  static final Serializer<Value> serializer = _$serializer;
+  bool get aBool;
+  @nullable double get aDouble;
```

Comment:
you may want to consider a second nullable field to make sure the join code works as expected